src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  &--4 :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Here should be all the modifiers, not only for 4 stars

Reply:
sorry, can you give more explanation? i don't understand your comment

Reply:
you should reuse your logic from previous task. 
It means that we should be able to set another modifier from 0 to 5 to show a correct amount of stars